spec/windows/block_device.go

```diff
@@ -7,26 +7,31 @@ import (
 	"unsafe"
 
 	"github.com/mackerelio/mackerel-agent/logging"
-	. "github.com/mackerelio/mackerel-agent/util/windows"
+	"github.com/mackerelio/mackerel-agent/util/windows"
 )
 
+// BlockDeviceGenerator XXX
 type BlockDeviceGenerator struct {
 }
 
+// Key XXX
 func (g *BlockDeviceGenerator) Key() string {
 	return "block_device"
 }
 
 var blockDeviceLogger = logging.GetLogger("spec.block_device")
 
+// Generate XXX
 func (g *BlockDeviceGenerator) Generate() (interface{}, error) {
 	results := make(map[string]map[string]interface{})
 
 	drivebuf := make([]byte, 256)
-	_, r, err := GetLogicalDriveStrings.Call(
+	r, _, err := windows.GetLogicalDriveStrings.Call(
 		uintptr(len(drivebuf)),
 		uintptr(unsafe.Pointer(&drivebuf[0])))
-	if r != 0 {
+	if r == 0 {
+		e := windows.GetLastError.Call
```

Comment:
Don't we need `()` to get last error?

I think `err` variable here already has the result `GetLastError` as far as I've read from the doc?: https://github.com/golang/go/blob/e67a39e/src/syscall/dll_windows.go#L121-124